pkg/aggregator/context_resolver_test.go

```diff
@@ -14,10 +14,14 @@ import (
 	"github.com/stretchr/testify/require"
 
 	tagger "github.com/DataDog/datadog-agent/comp/core/tagger/def"
+	taggerfxmock "github.com/DataDog/datadog-agent/comp/core/tagger/fx-mock"
 	nooptagger "github.com/DataDog/datadog-agent/comp/core/tagger/impl-noop"
+	taggerTypes "github.com/DataDog/datadog-agent/comp/core/tagger/types"
 	"github.com/DataDog/datadog-agent/pkg/aggregator/ckey"
 	"github.com/DataDog/datadog-agent/pkg/aggregator/internal/tags"
+	"github.com/DataDog/datadog-agent/pkg/filterlist"
 	"github.com/DataDog/datadog-agent/pkg/metrics"
+	taggertypes "github.com/DataDog/datadog-agent/pkg/tagger/types"
```

Comment:
This the second import with the name `taggertypes` with different capitalization. Are these different packages? If yes, can we give them more distinct names?